src/aind_slims_api/core.py

```diff
@@ -204,6 +99,31 @@ def fetch(
 
         return records
 
+    def resolve_model_alias(
```

Comment:
This method can be static

Reply:
resolved in [c353219](https://github.com/AllenNeuralDynamics/aind-slims-api/pull/20/commits/c353219a49206068eefaad25f4f75c3fc9cc22e9)